common/src/main/java/com/wynntils/core/webapi/profiles/TerritoryProfile.java

```diff
@@ -164,16 +167,21 @@ public TerritoryProfile deserialize(JsonElement json, Type typeOfT, JsonDeserial
             }
 
             String guildPrefix;
-            if (territory.get("guildPrefix").isJsonNull()) guildPrefix = "UNK";
-            else guildPrefix = territory.get("guildPrefix").getAsString();
+            if (territory.get("guildPrefix").isJsonNull()) {
+                guildPrefix = "UNK";
```

Comment:
Could this be an actual guild prefix? Maybe a prefix of UNKWN or something would be better

Reply:
I know it differs from legacy

Reply:
Fixed, shouldnt happen though.